Source/sources/thermo_nd.F90

```diff
@@ -41,19 +61,21 @@ subroutine ca_thermo_src(lo, hi, &
     do k = lo(3), hi(3)
        do j = lo(2), hi(2)
           do i = lo(1), hi(1)
-             src(i,j,k,UEINT) = FOURTH*((old_state(i+1,j,k,UMX)/old_state(i+1,j,k,URHO) + &
-                                         new_state(i+1,j,k,UMX)/new_state(i+1,j,k,URHO)) - &
-                                        (old_state(i-1,j,k,UMX)/old_state(i-1,j,k,URHO) + &
-                                         new_state(i-1,j,k,UMX)/new_state(i-1,j,k,URHO)))/dx(1)
+
+             ! compute -div{U}
+             src(i,j,k,UEINT) = -FOURTH*((old_state(i+1,j,k,UMX)/old_state(i+1,j,k,URHO) + &
```

Comment:
Unrelated to this PR, but is this calculation correct in non-Cartesian coordinates?

Reply:
this is now fixed